core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerFacade.java

```diff
@@ -123,6 +124,11 @@ long positionInAdvertBreakInMillis() throws IllegalStateException {
         return 0;
     }
 
+    private boolean isSetToPlayAdvert() {
```

Comment:
Do we still need this one? I think we can now simply inline it?, anyway, not a blocker, it's a private method so...

Reply:
I kept it because it's used twice, breaking the triple-usage rule 😄 YOLO